src-built-in/components/userPreferences/src/components/general/ScheduledRestart.jsx

```diff
@@ -0,0 +1,199 @@
+import React from 'react';
```

Comment:
@carterbs given the short deadlines for his code, fine, but this is technical debt. What we want is a generic, reusable way to display a "time" type preference rather than the first of soon to be a million hard coded preferences.

@jimbunting technical debt.